chapters/5-processing.tex

```diff
@@ -322,6 +324,10 @@ \section{Data quality assurance}
 Data quality assurance requires a combination of real-time data checks
 and back-checks or validation audits, which often means tracking down
 the people whose information is included in the dataset.
+Whether your team is conducting a survey or 
+you are receiving data from a partner,
```

Comment:
```suggestion
you are receiving administrative data from a partner,
```
To make it more clear that is what we mean, otherwise it sounds as we just as well could mean survey data from a partner